packages/gestalt/src/TableRowExpandable.js

```diff
@@ -43,10 +55,38 @@ export default function TableRowExpandable(props: Props): Node {
     }
   };
 
+  useEffect(() => {
+    if (rowRef && rowRef.current && stickyColumns) {
+      const colWidths = [];
+      const tableRowChildrenArray = [...rowRef.current.children];
+      tableRowChildrenArray.forEach((child) => {
```

Comment:
Same feedback here re: `.map`

Reply:
I wonder if some of this repeated logic could be extracted out into a helper file.